src/app/services/communibee-backend/volunteering-events/volunteering-event.ts

```diff
@@ -0,0 +1,33 @@
+import BackendModel from '../backend-model';
+
+export interface VolunteeringRequestOfferBase {
```

Comment:
We've changed this model:
https://github.com/CommuniBee/CommuniBee-Back-End/pull/19/files#diff-0b0983bbdae113a9dc90949652347237